UnityProject/Assets/Resources/ScriptableObjects/FloorSound/DefaultFloorSound.asset

```diff
@@ -14,31 +14,31 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
   Barefoot:
   - SetLoadSetting: 0
-    AssetAddress: Assets/Prefabs/footsteps/floor/hardbarefoot1.prefab
+    AssetAddress: null
```

Comment:
looks like you accidentally made the barefoot sounds null

Reply:
how did I even do that, what